examples/widgets/drag_and_drop.enaml

```diff
@@ -97,13 +98,14 @@ enamldef Main(Window):
         pass
     Container:
         constraints = [
-            hbox(vbox(lbl1, lbl2, spacer), target),
+            hbox(vbox(lbl1, lbl2), target),
         ]
         DragLabel: lbl1:
             text = 'Drag Me 1'
-            data = 'small\ndata'
+            data = b'small\ndata'
```

Comment:
With the latest fix from #18 (this branch has been rebased on it) I think everything now works as expected.